### PR TITLE
Adding author profile for Kaitlin Burns

### PR DESCRIPTION
**Pull request by KBurns073**
Adding author id for Kaitlin Burns

### CODE REVIEW

File: _posts/blog/2018-10-11-4rios-review.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "QUEERING THE DIGITAL ARCHIVE: '4 RIOS' AND GRAPHIC NARRATION"
+modified:
+categories: blog
+excerpt: ...the unique construction of this archive calls into question institutionalized norms regarding what an archive is and how it functions.
+tags: []
+author: "Kaitlin Burns"
```
Comment (halperta) [inline _posts/blog/2018-10-11-4rios-review.md]: The "author" should be your username as you entered it into the authors.yml file (i.e. kburns073)

File: _posts/blog/2018-10-11-4rios-review.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+image:
+	feature:
+---
+# **Queering the Method: "4 Ríos" and Archival Narration**
+
+4 Ríos Digital Archive. http://4rios.co. Acccessed 28 Sept. 2018.
```
Comment (halperta) [inline _posts/blog/2018-10-11-4rios-review.md]: Please make the url a clickable link using markdown formatting. Instructions: http://www.halperta.com/criticalarchives/tutorial/writing-in-markdown/

File: _posts/blog/2018-10-11-4rios-review.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+The stated goal of "4 Ríos" is to "propose other forms of approaching memory, to the construction of spaces of exchange and debate, to raise awareness about these facts, to not forgetting, to the rescue of that which, through the passing of time, is believed to be lost" ([Proyecto](http://4rios.co/blog/proyecto/), *my translation*). This is achieved by recounting experiences, facts, and the background and succession of evens in recent history of armed conflict in Colombia - specifically the 2001 massacre on the river Naya.
```
Comment (halperta) [inline _posts/blog/2018-10-11-4rios-review.md]: I found the first sentence confusing. Consider: "propose other forms of approaching memory, of constructing spaces... , of raising awareness... , of not forgetting, and of rescuing..."

File: _posts/blog/2018-10-11-4rios-review.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+On the opening webpage of "4 Ríos", the user's inital reaction is that of curious confusion, since the archive is not immediately recognizable as such. One might feel disillusioned by prior experiences with prototyical digital collections, library-like in their organization, as one attempts to master the interface and search for specific documentation. A visitor to "4 Ríos" transforms their experience when they agree to surrender instead, becoming a spectator as the narrative unfolds in graphic novel style, subject to the creators' conceived succession of information flow. If my description seems beyond grasp, it is because one must participate in this interactive historical retelling to conceive of its form.
```
Comment (halperta) [inline _posts/blog/2018-10-11-4rios-review.md]: This is lovely. 

I'm not sure "disillusioned" is the right word. Maybe "far from prior experiences..."?

File: _posts/blog/2018-10-11-4rios-review.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+A tap to the smaller circle at the left (["Flujo de Memorias"](http://4rios.co/elnaya/flujo)) brings one into a vibrant forum of expression, where a participant can leave thoughts for future viewers by drawing a picture, or leaving a written or recorded message. A swipe of the mouse reveals that the colorful dots flowing across the stream are the river of memories left by past users, and exploring these fragments grants another dimension to archival information. To the right ([" Plataformas y Expansiones"](http://4rios.co/blog/expansiones-y-plataformas/)), the visitor can read about further projects for dissemination of this information, including access to the printed version of the graphic novel. A menu bar on the top gives options to visit the (["4 Ríos" blog](http://4rios.co/blog)), team information (["Equipo"](http://4rios.co/blog/equipo)), information on the project's production (["¿Qué es 4 Ríos"](http://4rios.co/blog/proyecto)) which leads to yet another page with more in-depth options, and recognitions (["Reconocimientos"](http://4rios.co/blog/reconocimientos)).
```
Comment (halperta) [inline _posts/blog/2018-10-11-4rios-review.md]: I'm not sure we need a detailed description of each tab. Instead, consider describing more generally the ways in which this site does or does not serve different kinds of users. As a scholar, what kind of information can I find here? What's missing? What if I'm not a scholar - is this archive useful for other kinds of audiences?

File: _posts/blog/2018-10-11-4rios-review.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+All criticisms aside, the unique construction of the "4 Ríos" archive calls into question institutionalized norms regarding what an archive is and how it functions. It provides an example of Zeb Tortorici's notion of *queering the archive*, wherein the researcher must adopt unconventional methods to tell the story that is usually left out of history. He states, "*queer archivalism*, in the words of Elizabeth Freeman - allows us to explore the fraught (and anachronistic) relations between past and present, archive and the document, historian and witness, writer and written of, consumer and consumed" (2018, 15-16). The "4 Ríos" collection guides it participants through the narration of events, facilitating education on the massacre of the Naya while providing access to important documentation through online links and video testimony, and questioning beaurocratic institutions of control.
```
Comment (halperta) [inline _posts/blog/2018-10-11-4rios-review.md]: I like how you bring in Tortorici here.

**Review by halperta — CHANGES_REQUESTED** [review]
Thanks for this review, Kaitlin. I particularly like your description of the affective experience of moving through the site. You'll receive more detailed feedback from me and Kelly after publication. For now, here are some editorial comments. Please take the next week to make any changes you'd like. Let me know if you have any questions!

File: _posts/blog/2018-10-11-4rios-review.md (hunk 3 above)
Comment (KBurns073) [reply _posts/blog/2018-10-11-4rios-review.md]: thanks! sometimes its hard to get outside of a translation and read it without the original voice!

Comment (KBurns073) [reply _posts/blog/2018-10-11-4rios-review.md]: do you think there is any problem with changing the original syntax? The original phrase is a bit convoluted itself...

Comment (KBurns073) [reply _posts/blog/2018-10-11-4rios-review.md]: The stated goal of "4 Ríos" is to "propose other forms of approaching memory, to construct spaces of exchange and debate, to raise awareness about these facts, to not forget, to rescue that which, through the passing of time, is believed to be lost" 

Does this work better? if you break it down, it now reads: the goal is to propose, to construct, to raise awareness, etc

File: _posts/blog/2018-10-11-4rios-review.md (hunk 4 above)
Comment (KBurns073) [reply _posts/blog/2018-10-11-4rios-review.md]: do you think "estranged" works? i just don't want to get too wordy since it's already a bit of a long sentence...

File: _posts/blog/2018-10-11-4rios-review.md (hunk 3 above)
Comment (halperta) [reply _posts/blog/2018-10-11-4rios-review.md]: This works a lot better! Excellent.

File: _posts/blog/2018-10-11-4rios-review.md (hunk 4 above)
Comment (halperta) [reply _posts/blog/2018-10-11-4rios-review.md]: I like estranged!

**Review by halperta — APPROVED** [review]
These changes all look great.

**Comment by KBurns073** [comment]
Hannah,

They look great! Quick question: am I able to continue editing in the way I have been editing bit by bit? I was thinking about doing something different with the titles.

Thanks,
Kaitlin

Kaitlin A Burns
Assistant Instructor, PhD Student
Department of Spanish and Portuguese
University of Texas at Austin
